File: Java Labs/Patterns On Java/src/com/pattern/adapter/InventoryItem.java
package com.pattern.adapter;

public class InventoryItem {
    private String name;
    private String itemColor;
    private int itemSize;

    public InventoryItem(String name, String itemColor, int itemSize) {
        this.name = name;
        this.itemColor = itemColor;
        this.itemSize = itemSize;
    }

    public String getName() {
        return name;
    }

    public String getItemColor() {
        return itemColor;
    }

    public int getItemSize() {
        return itemSize;
    }
}

File: Python Labs/com/pattern/adapter/InventoryItem.py
class InventoryItem:
    def __init__(self, name, item_color, item_size):
        self.name = name
        self.item_color = item_color
        self.item_size = item_size

    def get_name(self):
        return self.name

    def get_item_color(self):
        return self.item_color

    def get_item_size(self):
        return self.item_size